debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -111,6 +111,72 @@ public static HStoreHandlingMode parse(String value, String defaultValue) {
         }
     }
 
+    /**
+     * The set of predefined BinaryHandlingMode options or aliases
+     */
+    public enum BinaryHandlingMode implements EnumeratedValue {
```

Comment:
This should go to `CommonConnectorConfig`.

Reply:
DONE